crates/astria-core/src/primitive/v1/mod.rs

```diff
@@ -488,6 +488,103 @@ where
     tree
 }
 
+#[derive(Debug, Clone, PartialEq, Eq, Hash, PartialOrd, Ord)]
+#[cfg_attr(feature = "serde", derive(serde::Serialize))]
+#[cfg_attr(feature = "serde", serde(transparent))]
```

Comment:
This should serialize like the protobuf type:

```suggestion
#[cfg_attr(
    feature = "serde",
    serde(try_from = "raw::TransactionId", into = "raw::TransactionId"),
)]
```